README.md

```diff
@@ -49,7 +49,7 @@ Per default the locale is set to english `en`.
 
 Per default the word per minute is set to `300`.
 
-At the moment there is only 12 supported locales: `en`, `fr`, `es`, `pt-br`, `cn`, `ja`, `de`, `tr`, `ro`, `bn`, `sk` and `cs`.
+At the moment there is only 12 supported locales: `en`, `fr`, `es`, `pt-br`, `zh-cn`, `zh-tw`, `ja`, `de`, `tr`, `ro`, `bn`, `sk` and `cs`.
```

Comment:
I'm thinking we could remove the `X` (number) since more people are contributing and it's hard to keep track of that number

```suggestion
At the moment it supports these locales: `en`, `fr`, `es`, `pt-br`, `zh-cn`, `zh-tw`, `ja`, `de`, `tr`, `ro`, `bn`, `sk` and `cs`.
```